data/reusables/gpg/copy-ssh-public-key.md

```diff
@@ -20,12 +20,20 @@
    $ clip &lt; ~/.ssh/id_{% ifversion ghae %}rsa{% else %}ed25519{% endif %}.pub
    # Copies the contents of the id_{% ifversion ghae %}rsa{% else %}ed25519{% endif %}.pub file to your clipboard
    ```
-
-   {% tip %}
-
-   **Tip:** With Windows Subsystem for Linux (WSL), you can use `clip.exe`. Otherwise if `clip` isn't working, you can locate the hidden `.ssh` folder, open the file in your favorite text editor, and copy it to your clipboard.
-
-   {% endtip %}
+   {% note %}
+   
+   **Notes:**
+   
+   - With Windows Subsystem for Linux (WSL), you can use `clip.exe`. Otherwise if `clip` isn't working, you can locate the hidden `.ssh` folder, open the file in your favorite text editor, and copy it to your clipboard.
+   - On newer versions of Windows that use the Windows Terminal, or anywhere else that uses the PowerShell command line, you may receive a `ParseError` stating that `The '&lt;' operator is reserved for future use.` In this case, the following alternative `clip` command should be used:
+   
+      ```shell
+      $ cat ~/.ssh/id_{% ifversion ghae %}rsa{% else %}ed25519{% endif %}.pub | clip
+      # Copies the contents of the id_{% ifversion ghae %}rsa{% else %}ed25519{% endif %}.pub file to your clipboard
+      ```
```

Comment:
I believe the indentation just needs to be moved to the left 3 spaces. It should be aligned with the bulleted points:

![Screenshot 2023-08-08 at 5 21 46 PM](https://github.com/github/docs/assets/22055162/249751e7-07b7-4051-8e8b-2f33a4921e46)

I believe that should fix the rendering issue!

Reply:
Great, thanks @gracepark! That is certainly an easy fix. 😄 

@stevecat I will make the changes and commit.

Reply:
@gracepark Mine on my fork looks a little different wrt to the indentation, i.e. the code inside the \``` shell fence is not indented, but the entire \``` shell code-fence is indented:

![image](https://github.com/github/docs/assets/1829396/f6ac30ef-20f7-430e-9e08-d91a9ad46520)

Reply:
So I committed a fix for the indentation of the entire /```shell code fence: b043740ecb57c93d66b3249018459a15419882cd

Please let me know if that is the correct fix or if you need me to make further changes.